### PR TITLE
Add git to rd edition (fixes #83)

### LINKED ISSUE #83: Add git to rd edition
See: rancher-sandbox/rancher-desktop#2771

This could be implemented as a `LIMA_INSTALL_GIT` option used by the "rd" edition which adds the `git` apk and any dependencies.

### PR DESCRIPTION
**Pull request by ryanfb**

### CODE REVIEW

**Comment by AkihiroSuda** [comment]
Thanks, but please sign the commit for DCO
https://github.com/apps/dco

(run `git commit -a -s --amend`, and make sure that the `Signed-off-by: NAME <EMAIL>` line with your real name is included in the commit message)

File: edition/rd [hunk 1]
```diff
@@ -6,6 +6,7 @@ LIMA_INSTALL_CNI_PLUGIN_FLANNEL=true
 LIMA_INSTALL_CURL=true
 LIMA_INSTALL_DOCKER=true
 LIMA_INSTALL_CRI_DOCKERD=true
+LIMA_INSTALL_GIT=true
```
Comment (AkihiroSuda) [inline edition/rd:9]: Could you update other edition files too?
I guess we can safely install git for all editions (except `min`)

cc @mook-as

Comment (ryanfb) [reply edition/rd:9]: Should now be done in https://github.com/lima-vm/alpine-lima/pull/84/commits/8acde0f04ec55c40ba3a5905c08490ccb9df2de3

Comment (ryanfb) [reply edition/rd:9]: Apologies for the confusion, not sure why the previous commit didn't get a signed-off-by, but I've now re-pushed it with a signature at 00ab4033ee737c0dc8c7a50383c1f1987b7b25ae

**Review by AkihiroSuda — APPROVED** [review]
Thanks

cc @jandubois

**Pull request merged**